var/spack/repos/builtin/packages/audacious/package.py

```diff
@@ -22,8 +22,10 @@ class Audacious(AutotoolsPackage):
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool',  type='build')
-    depends_on('glib')
-    depends_on('qt')
+    depends_on('gettext',  type=('build', 'link'))
+    depends_on('iconv',    type='link')
+    depends_on('glib',     type='link')
+    depends_on('qt',       type='link')
```

Comment:
Can we remove types from these dependencies and use the default, which seems to fit? If instead we want for some reason to stick with these, isn't `qt` a `build,link` dependency due to the presence of MOC and similar machinery?